docs/guides/kubernetes.md

```diff
@@ -80,160 +72,108 @@ subjects:
-    metadata:
-      labels:
-        app: pomerium
-    spec:
-      containers:
-        - name: pomerium
-          image: pomerium/pomerium:master
-          ports:
-            - containerPort: 30443
-          env:
-            - name: ADDRESS
-              value: "0.0.0.0:30443"
-            - name: AUTHENTICATE_SERVICE_URL
-              value: "https://authenticate.localhost.pomerium.io:30443"
-            - name: CERTIFICATE
-              value: "..." # $(base64 -w 0 <./_wildcard.localhost.pomerium.io.pem)
-            - name: CERTIFICATE_KEY
-              value: "..." # $(base64 -w 0 <./_wildcard.localhost.pomerium.io-key.pem)
-            - name: COOKIE_SECRET
-              value: "..." # $(head -c32 /dev/urandom | base64 -w 0)
-            - name: IDP_PROVIDER
-              value: google
-            - name: IDP_CLIENT_ID
-              value: "..."
-            - name: IDP_CLIENT_SECRET
-              value: "..."
-            - name: POLICY
-              value: "..." #$(echo "$_policy" | base64 -w 0)
+    ```yaml
+      routes:
+        - from: https://k8s.localhost.pomerium.io
+          to: https://kubernetes.default.svc.cluster.local
+          allow_spdy: true
+          tls_skip_verify: true
+          kubernetes_service_account_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+          policy:
+            - allow:
+                or:
+                  - domain:
+                      is: pomerium.com
+    ```
 
----
-apiVersion: v1
-kind: Service
-metadata:
-  namespace: default
-  name: pomerium
-spec:
-  type: NodePort
-  selector:
-    app: pomerium
-  ports:
-    - port: 30443
-      targetPort: 30443
-      nodePort: 30443
-```
+    Change the policy to match your configuration.
 
-Make sure to fill in the appropriate values as indicated.
+1. Apply the updated values with Helm:
 
-The policy should be a base64-encoded block of yaml:
+    ```bash
+    helm upgrade --install pomerium pomerium/pomerium --values pomerium-values.yaml
+    ```
 
-```yaml
-- from: https://k8s.localhost.pomerium.io:30443
-  to: https://kubernetes.default.svc
-  tls_skip_verify: true
-  allow_spdy: true
-  policy:
-    - allow:
-        or:
-          - domain:
-              is: pomerium.com
-  kubernetes_service_account_token: "..." #$(kubectl get secret/"$(kubectl get serviceaccount/pomerium -o json | jq -r '.secrets[0].name')" -o json | jq -r .data.token | base64 -d)
-```
+    This will create a Pomerium route within kubernetes that is accessible from `*.localhost.pomerium.io`.
 
-Applying this configuration will create a Pomerium deployment and service within kubernetes that is accessible from `*.localhost.pomerium.io:30443`.
+## Configure Kubectl
 
-## Kubectl
+The [pomerium-cli] tool can be used by kubectl as an auth helper. Once configured, connections to the cluster will open a browser window to the Pomerium authenticate service and generate an authorization token that will be used for Kubernetes API calls.
```

Comment:
```suggestion
The [pomerium-cli] tool can be used by kubectl as an authentication helper. Once configured, connections to the cluster will open a browser window to the Pomerium authenticate service and generate an authorization token that will be used for Kubernetes API calls.
```

Reply:
CC @travisgroth for what term is more accepted in the k8s space. Google search results favor "auth":
![image](https://user-images.githubusercontent.com/2349184/144089980-1c693bb4-6d79-47b8-9f78-96d9db1d5c38.png)
![image](https://user-images.githubusercontent.com/2349184/144090036-28178ca3-d927-4656-95dc-1564fc7781bf.png)

Reply:
I'm not sure either is strictly accurate.  Maybe best to rephrase a bit.  According to https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins, we're technically a credential plugin.

Reply:
```suggestion
The [pomerium-cli] tool can be used by kubectl as credential plugin. Once configured, connections to the cluster will open a browser window to the Pomerium authenticate service and generate an authentication token that will be used for Kubernetes API calls.
```